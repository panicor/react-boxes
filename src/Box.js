import React from "react";

function Box({ id, handleRemove, width, height, backgroundColor }) {
  let removeBox = () => handleRemove(id);
  return <div>
    <div style ={{height: `${height}px`, width: `${width}px`, backgroundColor}}>
    </div>
    <button onClick={removeBox}> Remove</button>
  </div>;
}

export default Box;
