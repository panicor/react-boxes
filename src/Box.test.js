import React from "react";
import { render } from '@testing-library/react';
import Box from './Box';

test('renders', () => {
  render(<Box />);
});

test("snapshot matches", () => {
  let {asFragment} = render(<Box />);
  expect(asFragment()).toMatchSnapshot();
})