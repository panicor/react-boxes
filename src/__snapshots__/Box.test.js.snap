// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot matches 1`] = `
<DocumentFragment>
  <div>
    <div />
    <button>
       Remove
    </button>
  </div>
</DocumentFragment>
`;
